ACE/ace/config-linux-common.h

```diff
@@ -173,7 +173,7 @@
 
 // Starting with FC9 rawhide this file is not available anymore but
 // this define is set
-#if defined _XOPEN_STREAMS && _XOPEN_STREAMS == -1
+#if (defined(__GLIBC__) && (__GLIBC__ == 2 && __GLIBC_MINOR__ >= 30)) || (defined _XOPEN_STREAMS && _XOPEN_STREAMS == -1)
```

Comment:
```suggestion
#if !defined(_XOPEN_STREAMS) || (defined _XOPEN_STREAMS && _XOPEN_STREAMS == -1)
```

Reply:
Agreed with suggested changes, as _XOPEN_STREAMS in not defined in newer release. If everyone is agree, I can send v2 patch.

Reply:
You should be able to commit my suggestion which automatically updates this PR